pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -135,8 +135,13 @@ func niProbingUpdatePort(ctx *zedrouterContext, port types.NetworkPortStatus,
 		if info, ok := netstatus.PInfo[port.IfName]; ok {
 			log.Infof("niProbingUpdatePort:   info intf %s is present %v\n", info.IfName, info.IsPresent)
 		}
+		if netstatus.CurrentUplinkIntf == "" { // assign the CurrentUplinkIntf even for non-Mgmt port
```

Comment:
Would it be hard to check if we have local IP addresses for the non-management ports?